src/api/models/index.js

```diff
@@ -8,6 +8,9 @@ const sequelize = new Sequelize(env.dbUrl, {
 
 const models = {
   User: sequelize.import('./user'),
+  Token: sequelize.import('./token'),
```

Comment:
we no longer have the token model

Reply:
so I should delete it right?I am not using it I just rebased.